packages/toolpad-app/src/toolpad/AppEditor/HierarchyExplorer/index.tsx

```diff
@@ -244,6 +251,19 @@ export default function HierarchyExplorer({ appId, className }: HierarchyExplore
     [dom, domApi],
   );
 
+  const handlePageSettingsNode = React.useCallback(
+    (nodeId: NodeId) => {
+      if (nodeId) {
```

Comment:
We might not need to check if `nodeId` is truthy here? At least I don't see it in the other functions - I think it should always be defined?

Reply:
And in case it can be `undefined` then let's express it in the types as well.

Reply:
@apedroferreira I think you are right that we don't need to check if `nodeId` is truthy here -- I've removed this in https://github.com/mui/mui-toolpad/pull/1672/commits/a87f8b75e4caff4fe38acbedd884eed0ba8d304f